### ISSUE TITLE
Game freezing and crashing when going through the OW portal

### ISSUE THREAD

**Issue by cardinalsrulejd** [FTBTeam/FTB-Interactions#474]
## Modpack Version: 
1.3.0

## Bug Report:
When walking through the OW portal it will activate and then the game will freeze up and ultimately crash.

## Expected Behavior/Suggestions:
Game crashes when going through OW portal

## Is It Repeatable? Steps to Reproduce: 

1. This has happened at random but frequent times. Theres nothing I know of to be able to repeat it.
2. The crash log (below) is always the same.

## Link to Log or Crash File Paste**: 
https://pastebin.com/62Zxf1Kf

**Comment by Vyraal1** [comment]
Can you try updating to the latest Java first? Just to set the boiler plate before anything else, report says you're still on Java 8.0.51

**Comment by cardinalsrulejd** [comment]
I will update it. Just for the record I have never had this issue with the Nether portal.

**Comment by cardinalsrulejd** [comment]
New crash report after updating Java https://pastebin.com/bALLeLws

**Comment by cardinalsrulejd** [comment]
New crash log after updating Java https://pastebin.com/TpRRQmzR

**Comment by cardinalsrulejd** [comment]
After doing some testing by bypassing the OW portal, I had the same crash when I /tpdim 100 into the overworld. So its something to do with the going to the overworld and not the portal itself.

**Comment by cardinalsrulejd** [comment]
Log during the crash https://pastebin.com/hnL2NuGC

**Comment by Saereth** [comment]
generic code chicken lib crash makes no sense, we can pass it on to CCL and hope for a fix/response

**Comment by cardinalsrulejd** [comment]
I thought I would update this issue, because I just recently got to space and started having the same problem as I did when going through the OW portal. Its the exact same crash as I was getting. When I was looking at the logs again I noticed that codechickenlib World Extension Manager was being thrown as part of the crash log. I'm no coder, but was wondering if this could be a source of the issue that I'm having since its popping up again as I travel to new dimensions. Thanks for any help.
